content/intro-to-storybook/angular/en/screen.md

```diff
@@ -321,9 +324,75 @@ Cycling through states in Storybook makes it easy to test we’ve done this corr
   />
 </video>
 
+## Interactive stories
+
+So far, we've been able to build a fully functional application from the ground up, starting from a simple component up to a screen, continuously testing each change using our stories. But with each story added, we would also need to check each variation manually to ensure the UI doesn't break, which requires extra work.
```

Comment:
```suggestion
So far, we've been able to build a fully functional application from the ground up, starting from a simple component up to a screen and continuously testing each change using our stories. But each new story also requires a manual check on all the other stories to ensure the UI doesn't break. That's a lot of extra work.
```

Reply:
Fixed with the latest commit for all frameworks.